target/linux/ipq806x/files-5.4/arch/arm/boot/dts/qcom-ipq8065-xr500.dts

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+#include "qcom-ipq8065-nighthawk.dtsi"
+
+/ {
+	model = "Netgear XR500";
+	compatible = "netgear,xr500", "qcom,ipq8065", "qcom,ipq8064";
+	
+	aliases {
+		led-boot = &led_power_white;
+		led-failsafe = &led_power_amber;
+		led-running = &led_power_white;
+		led-upgrade = &led_power_amber;
+	};
+
+	leds {
+		
+		led_power_white: power_white {
+			label = "xr500:white:power";
+			gpios = <&qcom_pinmux 53 GPIO_ACTIVE_HIGH>;
+			default-state = "keep";
+		};
+
+		led_power_amber: power_amber {
+			label = "xr500:amber:power";
+			gpios = <&qcom_pinmux 9 GPIO_ACTIVE_HIGH>;
+		};
+
+		wan_white {
+			label = "xr500:white:wan";
+			gpios = <&qcom_pinmux 22 GPIO_ACTIVE_HIGH>;
+		};
+
+		wan_amber {
+			label = "xr500:amber:wan";
+			gpios = <&qcom_pinmux 23 GPIO_ACTIVE_HIGH>;
+		};
+		
+        usb1 {
+			label = "xr500:white:usb1";
+			gpios = <&qcom_pinmux 8 GPIO_ACTIVE_HIGH>;
+		};
+
+		usb2 {
+			label = "xr500:white:usb2";
+			gpios = <&qcom_pinmux 26 GPIO_ACTIVE_HIGH>;
+		};
+
+		esata {
+			label = "xr500:white:esata";
+			gpios = <&qcom_pinmux 7 GPIO_ACTIVE_HIGH>;
+		};
+		
+		wifi {
+			label = "xr500:white:wifi";
+			gpios = <&qcom_pinmux 64 GPIO_ACTIVE_HIGH>;
+		};
+
+		wps {
+			label = "xr500:white:wps";
+			gpios = <&qcom_pinmux 24 GPIO_ACTIVE_HIGH>;
+		};
+
+	};
+};
+
+&nand_controller {
+	nand@0 {
+		partitions {			
+			ubi@1880000 {
+				label = "ubi";
+				reg = <0x1880000 0xce00000>;
+			};
+
+			reserve@e680000 {
+				label = "reserve";
+				reg = <0xe680000 0x0780000>;
```

Comment:
That's only ee00000 (238 MiB) in total?

Reply:
as referenced n commit message - I've used forum patch.
Due to vendor specification there's 256MB flash. Using 238MB out of 256MB seems to be reasonable as there's UBI and remaning blocks might be used for readdress bad blocks in NAND flash.

Also checked ddwrt memory mappings and they are more strict ( https://svn.dd-wrt.com/browser/src/linux/universal/linux-4.9/arch/arm/boot/dts/qcom-ipq8065-xr500.dts )

```
reserved@e680000 {
label = "reserved";
reg = <0xe680000 0x6c0000>;
read-only;
};
```

Also I've tested commited setings for memory mapping, no any other.

Reply:
From partitioning I would have expected
`reg = <0xe680000 0x700000>;`
?